frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1146,6 +1178,9 @@ pub mod pallet {
 		Rewarded { account: <T as frame_system::Config>::AccountId, value: BalanceOf<T> },
 		/// An account has been slashed for submitting an invalid signed submission.
 		Slashed { account: <T as frame_system::Config>::AccountId, value: BalanceOf<T> },
+		/// The election failed but the emergency phase did not start because there was not enough
+		/// time to prepare the election results.
+		EmergencyPhaseThrottled,
```

Comment:
In the documentation above it has been mentioned
>  It is possible to set a minimum number of "electing" blocks before an election failure tries the fallback election

Is this Emergency phase throttling or more generally fallback throttling?